emubd/lfs_emubd.c

```diff
@@ -47,19 +47,27 @@ int lfs_emubd_create(const struct lfs_config *cfg, const char *path) {
 
     // Load stats to continue incrementing
     snprintf(emu->child, LFS_NAME_MAX, "stats");
-    FILE *f = fopen(emu->path, "r");
-    if (!f) {
-        return -errno;
-    }
 
-    size_t res = fread(&emu->stats, sizeof(emu->stats), 1, f);
-    if (res < 1) {
+    FILE *f = fopen(emu->path, "r");
+    if (!f && errno != ENOENT) {
         return -errno;
-    }
+    } else if (errno == ENOENT) {
+        f = fopen(emu->path, "wb");
```

Comment:
Thanks for all the changes :+1: 

One last note, we no longer need to open/close the file on this branch.